crates/host/src/wasmbus/mod.rs

```diff
@@ -3675,7 +3695,7 @@ impl Host {
 
     #[instrument(level = "trace", skip_all)]
     async fn handle_config_get_one(&self, entity_id: &str, key: &str) -> anyhow::Result<Bytes> {
-        trace!("getting config");
+        trace!("handling config");
```

Comment:
```suggestion
        trace!(%entity_id, "handling config");
```
To match the other one?